Model/Method/Capayable.php

```diff
@@ -257,9 +261,15 @@ private function getCustomerData($payment)
     {
         /**@var Address $billingAddress */
         $billingAddress = $payment->getOrder()->getBillingAddress();
+        $phone = $billingAddress->getTelephone();
+
+        if ($payment->getAdditionalInformation('customer_telephone') !== null) {
+            $phone = $payment->getAdditionalInformation('customer_telephone');
+        }
+        
         $now = new \DateTime();
         $phoneData = $this->addressFormatter->formatTelephone(
-            $billingAddress->getTelephone(),
+            $phone,
```

Comment:
I think solution from Andrei is good here, to send Telephone from billing if phone is still empty. 
(empty($telephone) ? $billingAddress->getTelephone() : $telephone),

Reply:
I miss this line: $phone = $billingAddress->getTelephone();